src/test/java/stepdefs/CucumberHook.java

```diff
@@ -0,0 +1,50 @@
+package stepdefs;
+
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.Selenide;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.Scenario;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import lombok.extern.slf4j.Slf4j;
+import org.openqa.selenium.OutputType;
+
+import java.util.Base64;
+
+@Slf4j
+public class CucumberHook {
+
+  @Before
+  public void beforeScenario(Scenario scenario) {
+    WebDriverManager.chromedriver().setup();
+
+    Configuration.browser = "chrome";
+    Configuration.driverManagerEnabled = true;
+    Configuration.browserSize = "1920x1080";
+    Configuration.headless = false;
+    Configuration.webdriverLogsEnabled = true;
+    log.info("START SCENARIO: " + scenario.getName());
+  }
```

Comment:
as a suggestion, you can move all of these properties to `selenide.properties` file and remove `beforeScenario` method at all as selenide will automatically take properties from those file

Reply:
ok, thanks, I'll investigate it, but this realization is more needed for adding the annotation)

Reply:
Done